src/services/StreamService.ts

```diff
@@ -8,9 +8,9 @@ class StreamService {
      * to a JS object
      * @param event
      */
-    public static getActivitiesStream(event: any) {
-        return event.Records.filter((record: DynamoDBRecord) => { // Retrieve "MODIFY" events
-            return record.eventName === "MODIFY";
+    public static getVisitsStream(event: any) {
```

Comment:
going forward tslint will error `any` . Can you derive a type from usage if quick and easy ... if not then please leave a comment for todo .. so this is picked as part of the typescript migration task

Reply:
This is a production bug, so I'd suggest the "todo" approach for now. We'll fix it up when it's not affecting prod.